src/zope/pagetemplate/engine.py

```diff
@@ -19,15 +19,19 @@
 
 import sys
 
-from zope import component
+from zope.i18n import translate
 from zope.interface import implementer
 from zope.interface.interfaces import ComponentLookupError
 from zope.proxy import isProxy
-from zope.traversing.interfaces import IPathAdapter, ITraversable
-from zope.traversing.interfaces import TraversalError
+from zope.security.proxy import ProxyFactory
+from zope.security.proxy import removeSecurityProxy
 from zope.traversing.adapters import traversePathElement
-from zope.security.proxy import ProxyFactory, removeSecurityProxy
-from zope.i18n import translate
+from zope.traversing.interfaces import IPathAdapter
+from zope.traversing.interfaces import ITraversable
+from zope.traversing.interfaces import TraversalError
+
+from zope import component
+
 
 try:  # pragma: no cover
     # Until https://github.com/zopefoundation/zope.untrustedpython/issues/2
```

Comment:
This issue is closed already, not sure if that means that this `try/except` can be removed though 🤔